### ISSUE TITLE
[Feature] Please add some things would be great

### ISSUE THREAD

**Issue by gselsidi** [Pythagora-io/pythagora#4]
1. Please add nextjs support and other DB's probably most used like mysql. 
2. Have it write negative tests, meaning failures so it can test: wrong responses, unathorized, ect.

**Comment by gselsidi** [comment]
this is what we have in our app btw:

`const app = next({ dev, dir: "./src" });
const server = express();`

I'm very interested in this, let me know if you add support for the above would be really interested in testing it out

**Comment by zvone187** [comment]
Definitely, we’re looking to expand support for more frameworks. Nextjs will likely be amongst first ones. We had a conversation about Nextjs on a [Reddit post](https://www.reddit.com/r/javascript/comments/10xwt0d/ive_created_an_open_source_tool_that_generates/j7vz00m/?utm_source=share&utm_medium=ios_app&utm_name=iossmf&context=3) .
MySQL will also be supported just not sure when exactly.

The code you mentioned works well with Pythagora. What you would need to add is just one line after initializing express. Like this:
```
const app = next({ dev, dir: "./src" });
const server = express();
if (global.Pythagora) global.Pythagora.addApp(server);
```

If you have a project with native express and Mongo, we’d love to hear how the integration went. If you need any help we’re happy to help.

**Comment by ManUtopiK** [comment]
Please, consider adding Nuxtjs, Fastify, postgresQL ! And integration with Vitest !

**Comment by zvone187** [comment]
Thanks for the suggestion - this helps us prioritize which tech should we support. Nuxtjs will actually be supported quite soon.

**Comment by Morphexe** [comment]
Vitest would be a godsend :)
